server/cli/launch.py

```diff
@@ -51,14 +51,14 @@ def annotation_args(func):
         "Incompatible with --annotations-input-file.",
     )
     @click.option(
-        "--experimental-annotations-ontology",
+        "--annotations-ontology",
```

Comment:
This should remain experimental, as we discussed and decided, it is not yet ready for non-experimental status.

Reply:
My understanding from our discussion was that the new ontologies module resolved the issues identified as blockers for removing the experimental tag. Apologies if I've misunderstood. @colinmegill @liaprins-czi -- What are your feelings on this?

Reply:
Definitely not - we explicitly discussed the need to look at hierarchies and other features previously outlined by Lia before we move to fully supported.   I also don't see the harm in leaving as is given that it is not impeding our pilot user.

Reply:
I think regular annotations is ready to leave the experimental realm, but I think there's more to do to make ontologies good enough (for example exploring more around the concept of revealing hierarchies and whether to/how to limit the label autosuggestions to children of the category parent, etc).

Reply:
Cool, thanks for clarifying. Will fix :)